src/css/dark.css

```diff
@@ -569,15 +562,11 @@ a.button:hover {
   float: left;
   overflow: hidden;
   display: inline-block;
-  -webkit-box-shadow: 3px 3px 8px -2px rgba(0,0,0,0.63);
```

Comment:
fwiw, `{-webkit,-moz}box-shadow` aren't included in the autoprefixed CSS. but the [support](http://caniuse.com/#search=box-shadow) is good, so it's fine.